presto-docs/src/main/sphinx/release/release-346.md

```diff
@@ -0,0 +1,93 @@
+# Release 346 (XX Nov 2020)
+
+## General Changes
+
+* Add support for `RANGE BETWEEN <value> PRECEDING AND <value> FOLLOWING` window frames. ({issue}`609`)
```

Comment:
Might be easier to read as

> `RANGE BETWEEN ... PRECEDING AND ... FOLLOWING`

Reply:
I agree, but this is specifically about the case where a value is provided (vs what we support today: UNBOUNDED and CURRENT ROW)